src/components/basic/Button/button.tsx

```diff
@@ -148,6 +148,14 @@ export const LightOutlinedButton = styled(Button)`
   border-radius: ${DV.sizes.baseRadius}px;
 `;
 
+export const LightOutlinedButtonDisabled = styled(Button)`
```

Comment:
Currently, we are using tailwind and index.scss file to style our components instead of styled component. So, please rewrite this component.